CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
 cmake_minimum_required(VERSION 3.10)
-project(OpenColorIO VERSION 2.0.0)
+
+project(OpenColorIO 
+	VERSION 2.0.0
+	HOMEPAGE_URL http://opencolorio.org/
+	LANGUAGES CXX C)
+
+# "dev" for master or "" for any official release
+set(OpenColorIO_VERSION_RELEASE_TYPE "dev")
+set(OpenColorIO_VERSION "${OpenColorIO_VERSION_MAJOR}.${OpenColorIO_VERSION_MINOR}.${OpenColorIO_VERSION_PATCH}${OpenColorIO_VERSION_RELEASE_TYPE}")
```

Comment:
The OpenColorIO_VERSION variable is set via the VERSION field of the project call.  Perhaps we should implement the "dev" flag in the OpenColorIO_VERSION_TWEAK attribute?

https://cmake.org/cmake/help/v3.10/command/project.html

Reply:
Unfortunately, ***_TWEAK*** only acceptes unsigned integers.